src/main/java/dev/sigstore/oidc/client/OidcException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 The Sigstore Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.sigstore.oidc.client;
+
+public class OidcException extends Exception {
```

Comment:
I guess we create these so that we can specialize the exceptions in the future?   I remember back in the day, the cool way to throw exceptions was to use a subtype of RuntimeException unless the caller was likely to be able to handle the issue.

Reply:
Yeah I do. lulz

Reply:
I meant runtime environment/programming error. Not the jdk.